packages/engine/src/avatar/systems/AvatarAnimationSystem.tsx

```diff
@@ -375,26 +333,45 @@ const Reactor = () => {
   return null
 }
 
+const AnimationReactor = () => {
+  const animations = [preloadedAnimations.locomotion, preloadedAnimations.emotes]
+
+  const loadedAnimations = useLoadAnimationFromBatchGLTF(
+    animations.map((animationFile) => {
+      return `${
+        getState(DomainConfigState).cloudDomain
+      }/projects/ir-engine/default-project/assets/animations/${animationFile}.glb`
+    }),
+    true
+  )
+
+  useEffect(() => {
+    if (!loadedAnimations.value) return
+    let i = 0
+    for (const loadedAnimationEntity of loadedAnimations.value as [AnimationClip[] | null, Entity][]) {
+      for (const animation of loadedAnimationEntity[0]!) {
+        retargetAnimationClip(animation, loadedAnimationEntity[1])
+        bindAnimationClipFromMixamo(animation)
+      }
+      getMutableState(AnimationState).loadedAnimations[animations[i]].set(loadedAnimationEntity[1]!)
+      i++
+    }
+  }, [loadedAnimations])
+  return null
+}
+
 export const AvatarAnimationSystem = defineSystem({
   uuid: 'ee.engine.AvatarAnimationSystem',
   insert: { after: AnimationSystem },
   execute,
   reactor: () => {
-    useEffect(() => {
-      const networkState = getMutableState(NetworkState)
-
-      networkState.networkSchema[IKSerialization.ID].set({
-        read: IKSerialization.readBlendWeight,
-        write: IKSerialization.writeBlendWeight
-      })
-
-      return () => {
-        networkState.networkSchema[IKSerialization.ID].set(none)
-      }
-    }, [])
-
-    if (!useMutableState(EngineState).viewerEntity.value) return null
```

Comment:
i think we still want to check that the viewer entity exists, or at least that there is a rendering context active. the original check here is to avoid loading these assets on pages that don't need them at load (such as the admin panel, which will use it if the user loads the avatar preview, but most of the time they don't). we could achieve this with maybe if a `useQuery([RendererComponent])`  is not empty